.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "eslint.enable": true,
```

Comment:
Remove the eslint for now, since we're going to be using tslint. I'll add the settings.json for tslint when I write our docku vscode extension.